niflow_manager/data/templates/base/spec.yml

```diff
@@ -0,0 +1,50 @@
+#  # requirement for the environment needed to run the analyses
```

Comment:
If possible, I would much prefer this to be a minimal, working thing. We can comment out suggestions for how to expand, but for things that will generally be used, let's just have it there.

Reply:
ok, changed the template and the test